about.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 
 <head>
-    <title>Lambda Theta Delta | Fraternity | Irvine</title>
+    <title>About | Lambda Theta Delta</title>
```

Comment:
Tailor every <title> tag to the specific page, improves SEO

Reply:
Was this the only page that needed this fix?

Reply:
No, this tittle should be made unique to each page
So the about page title would show case "About | Lambda Theta Delta"

Reply:
Oh, I was asking if the other pages had been updated, it looks like they have been, gonna merge this in now.